ui/src/views/CachePurges/ViewCachePurges/index.jsx

```diff
@@ -67,12 +72,17 @@ export default class ViewCachePurges extends Component {
     });
   };
 
+  handlePurgeCacheSubmit = cacheSearch => {
+    this.setState({ cacheSearch });
+  };
+
   render() {
     const {
       classes,
       description,
       data: { loading, error, cachePurges },
     } = this.props;
+    const { cacheSearch } = this.state;
```

Comment:
No need to use state. We can have access to the url directly. See how the hooks view does it for example:

https://github.com/taskcluster/taskcluster/blob/7113cdec74550c14d5f2a188f0bc9a047849386f/ui/src/views/Hooks/ListHooks/index.jsx#L55-L56

Reply:
Oh cool tip, that deep links it too.